build.gradle

```diff
@@ -1,12 +1,21 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.32"
+    ext {
+        kotlin_version = "1.6.10"
+        gradle_version = "7.0.4"
+        version_kotlinx_coroutines = "1.4.2"
```

Comment:
coroutines version should be 1.6.0, matching kotlin_version. I would also add a comment about this relationship and perhaps reorder gradle_version to the top.

I'm not 100% on what would happen when an implementing app has different version of kotlin and coroutines...